src/Microsoft.Health.Dicom.Api/Features/Exceptions/ExceptionHandlingMiddleware.cs

```diff
@@ -106,6 +106,7 @@ private IActionResult MapExceptionToResult(Exception exception)
             case InstanceAlreadyExistsException:
             case ExtendedQueryTagsAlreadyExistsException:
             case ExtendedQueryTagsOutOfDateException:
+            case ExistingUpdateOperationException:
```

Comment:
Can we add a test?

Reply:
Synced offline. We already have a test for the same.